internal/ent/hooks/orgmembers.go

```diff
@@ -193,3 +222,89 @@ func updateOrgMemberDefaultOrgOnCreate(ctx context.Context, m *generated.OrgMemb
 
 	return updateDefaultOrgIfPersonal(allowCtx, userID, orgID, m.Client())
 }
+
+func deleteSystemManagedUserGroup(ctx context.Context,
+	m *generated.OrgMembershipMutation, userID, orgID string) error {
+
+	user, err := m.Client().User.Get(ctx, userID)
+	if err != nil {
+		zerolog.Ctx(ctx).Error().Err(err).Msg("error getting user for managed group deletion")
+		return err
+	}
+
+	_, err = m.Client().Group.Delete().
+		Where(
+			group.CreatedBy(userID),
+			group.IsManaged(true),
+			group.OwnerID(orgID),
+			group.Name(getUserGroupName(user.DisplayName, user.ID)),
```

Comment:
can you set the display name to just the user display name?
```suggestion
			group.Name(getUserGroupName(user.DisplayName, user.ID)),
			group.DisplayName(user.DisplayName),
```